qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -52,20 +56,55 @@ class Calibrations:
     ScheduleBlock are supported.
     """
 
+    # The name of the parameter under which the qubit frequencies are registered.
+    __qubit_freq_parameter__ = "qubit_lo_freq"
+
+    # The name of the parameter under which the readout frequencies are registered.
+    __readout_freq_parameter__ = "meas_lo_freq"
+
     # The channel indices need to be parameterized following this regex.
     __channel_pattern__ = r"^ch\d[\.\d]*\${0,1}[\d]*$"
 
-    def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] = None):
+    def __init__(
+        self,
+        coupling_map: Optional[List[List[int]]] = None,
+        control_channel_map: Optional[Dict[Tuple[int, ...], List[ControlChannel]]] = None,
+        library: Optional[BasisGateLibrary] = None,
+        add_parameter_defaults: bool = True,
+    ):
         """Initialize the calibrations.
 
+        Calibrations can be initialized from a basis gate library, i.e. a subclass of
+        :class:`BasisGateLibrary`. As example consider the following code:
+
+        .. code-block:: python
+
+            cals = Calibrations(
+                    library=FixedFrequencyTransmon(
+                        basis_gates=["x", "sx"],
+                        default_values={duration: 320}
+                    )
+                )
+
         Args:
-            control_config: A configuration dictionary of any control channels. The
+            coupling_map: The device's coupling map. Each sub-list describes connected qubits
+                For example, the coupling map of a fully pairwise-connected backend with three
+                qubits is :code:`[[0, 1], [1, 0], [1, 2], [2, 1], [2, 0], [0, 2]]`.
+            control_channel_map: A configuration dictionary of any control channels. The
                 keys are tuples of qubits and the values are a list of ControlChannels
                 that correspond to the qubits in the keys.
+            library: A library class that will be instantiated with the library options to then
+                get template schedules to register as well as default parameter values.
+            add_parameter_defaults: A boolean to indicate weather the default parameter values of
+                the given library should be used to populate the calibrations. By default this
+                value is True but can be set to false when deserializing a calibrations object.
         """
 
+        # When True add_parameter_value triggers an inst. map update
+        self.update_inst_map_on_add = False
```

Comment:
This could be an option of `self.add_parameter_value` with default `True`? Seems like it is only used to bypass instmap update for the initial construction of calibration data for performance reason, so no need to have state.

Reply:
Good point. See here: 2e24e1e